meta/Meta.cpp

```diff
@@ -1783,11 +1783,10 @@ sai_status_t Meta::objectTypeGetAvailability(
 {
     SWSS_LOG_ENTER();
 
+    // When checking availability of a resource solely based on OBJECT_TYPE, attrCount is 0 and attrList is NULL
     PARAMETER_CHECK_OID_OBJECT_TYPE(switchId, SAI_OBJECT_TYPE_SWITCH);
     PARAMETER_CHECK_OID_EXISTS(switchId, SAI_OBJECT_TYPE_SWITCH);
     PARAMETER_CHECK_OBJECT_TYPE_VALID(objectType);
-    PARAMETER_CHECK_POSITIVE(attrCount);
-    PARAMETER_CHECK_IF_NOT_NULL(attrList);
```

Comment:
actually there should be one more check 
if (attrCount && attrList == NULL)
 // error, not expected nulll

Reply:
Done